Frontend/src/Components/Dashboard/Dashboard.js

```diff
@@ -453,6 +456,7 @@ function DataViewOptions(props) {
       <option style={{color: props.txtColor}} value={dvOptions.select}>Select option</option>
       <option style={{color: props.txtColor}} value={dvOptions.battery_pack}>BMS - Battery Pack</option>
       <option style={{color: props.txtColor}} value={dvOptions.cell_groups}>BMS - Cell Groups</option>
+      <option style={{color: props.txtColor}} value={dvOptions.battery}>Battery</option>

```

Comment:
```suggestion
      <option style={{color: props.txtColor}} value={dvOptions.battery}>BMS - Battery Pack</option>
```
Let's replace the previous "BMS - Battery Pack" with this new component. Also, you can delete the old `BatteryPack` component